### PR TITLE
Implement keyboard language selection popup

### LINKED ISSUE #1134: Implement keyboard language selection popup

### LINKED ISSUE #1150: Select Chinese Pinyin keyboard by default when user's OS locale is set as such
**Is your feature request related to a problem? Please describe.**
When the user's locale is `zh-TW`, should the Chinese Traditional (`zh-TW`) keyboard be shown by default? Currently, it shows the English keyboard. And the user has to know to click the Globe icon to change the keyboard.

**Describe the solution you'd like**
Ideally, on application launch FxR uses the keyboard corresponding to the system locale (i.ge, English vs. Pinyin today).

When the user changes the keyboard, FxR persists that changed system locale. Any subsequent relaunches use that system locale. The user could change it, etc.

**Describe alternatives you've considered**
Possibly leave it as is.

**Additional context**
This can be augmented by the work to be done in #1134.

### PR DESCRIPTION
**Pull request by MortimerGoro**
Fixes #1134. Fixes #1150.

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/ui/views/LanguageSelectorView.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
+package org.mozilla.vrbrowser.ui.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.util.TypedValue;
+import android.widget.Button;
```
Comment (cvan) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/views/LanguageSelectorView.java]: is this package used?

File: app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/KeyboardWidget.java [hunk 2]
```diff
@@ -451,6 +457,39 @@ public void swipeUp() {
...
+        // If the user has not selected any keyboard, find the best match from system locales
+        LocaleList localeList = getResources().getConfiguration().getLocales();
+        String[] supportedLocales = new String[mKeyboards.size()];
+        for (int i = 0; i < mKeyboards.size(); ++i) {
+            supportedLocales[i] = mKeyboards.get(i).getLocale().toLanguageTag();
+        }
+        Locale bestMatch = localeList.getFirstMatch(supportedLocales);
+        mCurrentKeyboard = getKeyboardForLocale(bestMatch);
+        if (mCurrentKeyboard == null) {
+            // Fallback to english.
```
Comment (cvan) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/KeyboardWidget.java]: nit: `Fallback to english,` -> `Fall back to English.`

File: app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/KeyboardWidget.java [hunk 3]
```diff
@@ -451,6 +457,39 @@ public void swipeUp() {
 
     }
 
+    private void setDefaultKeyboard() {
+        mCurrentKeyboard = getKeyboardForLocale(SettingsStore.getInstance(getContext()).getKeyboardLocale());
+        if (mCurrentKeyboard != null) {
+            return;
+        }
+
+        // If the user has not selected any keyboard, find the best match from system locales
```
Comment (cvan) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/KeyboardWidget.java]: nit: end with a period

**Comment by MortimerGoro** [comment]
@cvan Thanks for the review, I've fixed both problems

**Comment by cvan** [comment]
thanks, re-reviewing

**Comment by philip-lamb** [comment]
>     5. bug: the `Enter` key still appears in Mandarin when the `English (US)` keyboard is selected:
>        image

See issue #60. The blocker in Geckoview has been fixed.

**Pull request merged**